### PR TITLE
Priyanshi/rum android

### PR DESCRIPTION
**Pull request by priyanshi-gupta**
### What does this PR do?

Adds new menu items for RUM Android. Content sourced from dd-sdk-android
### Preview

https://docs-staging.datadoghq.com/priyanshi/rum_android/real_user_monitoring/android/

https://docs-staging.datadoghq.com/<BRANCH_NAME>/<PATH>

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: local/bin/py/build/configurations/pull_config_preview.yaml [hunk 1]
```diff
@@ -321,13 +321,57 @@
     - action: pull-and-push-file
       branch: master
       globs:
-      - 'docs/rum_collection.md'
+      - 'docs/rum_getting_started'
       options:
-        dest_path: '/real_user_monitoring/'
+        dest_path: '/real_user_monitoring/android'
```
Comment (kayayarai) [inline local/bin/py/build/configurations/pull_config_preview.yaml:326]:
```suggestion
        dest_path: '/real_user_monitoring/android/'
```

File: local/bin/py/build/configurations/pull_config_preview.yaml [hunk 2]
```diff
@@ -321,13 +321,57 @@
     - action: pull-and-push-file
       branch: master
       globs:
-      - 'docs/rum_collection.md'
+      - 'docs/rum_getting_started'
```
Comment (kayayarai) [inline local/bin/py/build/configurations/pull_config_preview.yaml:324]:
```suggestion
      - 'docs/rum_getting_started.md'
```

File: local/bin/py/build/configurations/pull_config_preview.yaml [hunk 3]
```diff
@@ -321,13 +321,57 @@
...
+          dependencies: ["https://github.com/DataDog/dd-sdk-android/blob/master/docs/rum_getting_started.md"]
+          title: RUM Android Monitoring
+          kind: documentation
+    - action: pull-and-push-file
+      branch: master
+      globs:
+      - 'docs/mobile_data_collected.md'
+      options:
+        dest_path: '/real_user_monitoring/android/data_collected'
+        file_name: 'mobile_data_collected.md'
```
Comment (kayayarai) [inline local/bin/py/build/configurations/pull_config_preview.yaml:338]:
```suggestion
        dest_path: '/real_user_monitoring/android/'
        file_name: 'data_collected.md'
```

File: local/bin/py/build/configurations/pull_config_preview.yaml [hunk 4]
```diff
@@ -321,13 +321,57 @@
...
+          dependencies: ["https://github.com/DataDog/dd-sdk-android/blob/master/docs/mobile_data_collected.md"]
+          title: RUM Android Data Collected
+          kind: documentation
+    - action: pull-and-push-file
+      branch: master
+      globs:
+      - 'docs/configure_rum_android_sdk.md'
+      options:
+        dest_path: '/real_user_monitoring/android/advanced_configuration'
+        file_name: 'configure_rum_android_sdk.md'
```
Comment (kayayarai) [inline local/bin/py/build/configurations/pull_config_preview.yaml:349]:
```suggestion
        dest_path: '/real_user_monitoring/android/'
        file_name: 'advanced_configuration.md'
```

File: local/bin/py/build/configurations/pull_config_preview.yaml [hunk 5]
```diff
@@ -321,13 +321,57 @@
...
+          dependencies: ["https://github.com/DataDog/dd-sdk-android/blob/master/docs/configure_rum_android_sdk.md"]
+          title: RUM Android Advanced Configuration
+          kind: documentation
+    - action: pull-and-push-file
+      branch: master
+      globs:
+      - 'docs/troubleshooting_android.md'
+      options:
+        dest_path: '/real_user_monitoring/android/troubleshooting'
+        file_name: 'troubleshooting_android.md'
```
Comment (kayayarai) [inline local/bin/py/build/configurations/pull_config_preview.yaml:360]:
```suggestion
        dest_path: '/real_user_monitoring/android/'
        file_name: 'troubleshooting.md'
```

File: local/bin/py/build/configurations/pull_config_preview.yaml [hunk 6]
```diff
@@ -321,13 +321,57 @@
...
+          dependencies: ["https://github.com/DataDog/dd-sdk-android/blob/master/docs/troubleshooting_android.md"]
+          title: Troubleshooting
+          kind: documentation
+    - action: pull-and-push-file
+      branch: master
+      globs:
+      - 'docs/integrated_libraries_android.md'
+      options:
+        dest_path: '/real_user_monitoring/android/integrated_libraries'
+        file_name: 'integrated_libraries_android.md'
```
Comment (kayayarai) [inline local/bin/py/build/configurations/pull_config_preview.yaml:371]:
```suggestion
        dest_path: '/real_user_monitoring/android/'
        file_name: 'integrated_libraries.md'
```

File: local/bin/py/build/configurations/pull_config_preview.yaml [hunk 7]
```diff
@@ -321,13 +321,57 @@
     - action: pull-and-push-file
       branch: master
       globs:
-      - 'docs/rum_collection.md'
+      - 'docs/rum_getting_started'
       options:
-        dest_path: '/real_user_monitoring/'
+        dest_path: '/real_user_monitoring/android'
         file_name: 'android.md'
```
Comment (kayayarai) [inline local/bin/py/build/configurations/pull_config_preview.yaml:327]:
```suggestion
        file_name: '_index.md'
```

**Review by kayayarai — CHANGES_REQUESTED** [review]
One missing `.md` extension, and fixing up the `dest_path` (which should only be folder names, the first two segments of the URL) and `file_name` (which should be the name of the content file in the docs site, not the source file name. It becomes the last segment of the URL)

File: local/bin/py/build/configurations/pull_config_preview.yaml (hunk 4 above)
Comment (priyanshi-gupta) [reply local/bin/py/build/configurations/pull_config_preview.yaml:349]: This file is called configure_rum_sdk.md in its original path -> would that be a problem?

https://github.com/DataDog/dd-sdk-android/blob/c80fef149ea92d3a6f63aa1a7487ea3a62d0b66e/docs/configure_rum_android_sdk.md

**Review by kayayarai — APPROVED** [review]

**Review by kayayarai — APPROVED** [review]

**Pull request merged**